openslides_backend/action/actions/motion_comment/create_delete_update.py

```diff
@@ -41,6 +43,27 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
         if allowed_groups.intersection(user_groups):
             return
 
+        motion_id = instance.get("motion_id")
+        if not motion_id:
+            comment = self.datastore.get(
+                fqid_from_collection_and_id(self.model.collection, instance["id"]),
+                ["motion_id"],
+            )
+            motion_id = comment.get("motion_id")
+
+        if (
+            section.get("submitter_can_write")
```

Comment:
Please check flag before reading the motion. The latter is not necessary, if the flag is false.